h2o-genmodel-extensions/xgboost/src/main/java/hex/genmodel/algos/xgboost/XGBoostMojoReader.java

```diff
@@ -37,29 +34,21 @@ protected XGBoostMojoModel makeModel(String[] columns, String[][] domains, Strin
     } catch (IOException e) {
       throw new IllegalStateException("MOJO is corrupted: cannot read the serialized Booster", e);
     }
-    Boolean conf = getJavaScoringConfig();
-    if (useJavaScoring(conf)) {
+    boolean javaScoringByDefault = readkv("use_java_scoring_by_default", false);
```

Comment:
this changes the behavior, old MOJOs that were predicting with java-predictor because they were using gbtree will start predicting with native

Reply:
good catch, fixed